src/components/tree-view/stories/tree.stories.tsx

```diff
@@ -102,3 +112,16 @@ export const Exemple: Story = {
     );
   },
 };
+
+export const WithViewMore: Story = {
```

Comment:
It's french translation, should be english

<img width="355" height="162" alt="Capture d’écran 2025-10-31 à 10 33 01" src="https://github.com/user-attachments/assets/554bf862-a6f7-47f4-bf0b-1350677a542d" />